.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ develop ]
```

Comment:
Does this mean the presumed default branch is 'develop'? I think some EBI steps (perhaps all) have the default usable branch as 'master'